src/components/core/Button/Button.tsx

```diff
@@ -49,7 +49,6 @@ const StyledButton = styled.button<StyledButtonProps>`
 
   .Button-label {
     opacity: 1;
-    transition: all ${transitions.cubic};
```

Comment:
there is already a transition defined on the parent so this was making the button-label text's transition occur after the parent transition, creating a 2-step transition effect when hovering on the button